notebooks/data-sources/TestGrid/metrics/unexpected_test_failures.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>can we have a count of the NA values? If it's 1% it wouldn't be a problem, if it's 99% we'd have to think this through.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Should the NA value counting go before the array is fillna'ed with zeros?</p>